saleor/core/prices.py

```diff
@@ -1,16 +1,18 @@
 from decimal import Decimal
-from typing import TYPE_CHECKING, Iterable, Union
+from typing import TYPE_CHECKING, Iterable, TypeVar
 
 from babel.numbers import get_currency_precision
 
 if TYPE_CHECKING:
     from django.db.models import Model
-    from prices import Money, TaxedMoney, TaxedMoneyRange
+    from prices import Money, TaxedMoney, TaxedMoneyRange  # noqa
```

Comment:
Why `# noqa`  here?

Reply:
`flake8` failed there, as it claimed that those types are unused, because they were used as forward references. I think we can just import them normally and remove those `# noqa`